app/components/signature-pill.js

```diff
@@ -89,14 +92,15 @@ export default class SignaturePillComponent extends Component {
   data = trackedTask(this, this.loadData, () => [this.triggerTask]); // Make the resource dependant on this.triggerTask
 
   get skin() {
-    const { REFUSED, CANCELED } = constants.SIGNFLOW_STATUSES;
     const statusUri = this.data.value?.status?.uri;
     const signingHubUrl = this.data.value?.signingHubUrl;
     if (statusUri === REFUSED || statusUri === CANCELED) {
       return 'error';
     } else if (signingHubUrl) {
-      return 'link';
+      return 'default';
+    } else if (statusUri === SIGNED) {
+      return 'success';
     }
-    return 'ongoing';
+    return 'link';
```

Comment:
when we have a link, we should have the skin link? (`"link"` is blue, `"default"` is grey)
So we need to show the blue link when we have a signingHubUrl or the route to signatures?

```
 const route = this.data.value?.route;
 
 .....
 
else if (signingHubUrl || route) {
   return 'link';
```

Reply:
Also, the css class `au-c-pill--link` shows a hover with a pointer. `au-c-pill-default` does not.